src/components/WwwFrame/WwwPageDesign.vue

```diff
@@ -26,26 +26,40 @@
 						</div>
 
 						<!-- nav for mobile and desktop -->
-						<ul class="
+						<nav id="top-nav">
+							<ul class="
 								tw-flex tw-gap-2 tw-items-center md:tw-justify-end tw-py-2.5
 								tw-bg-secondary md:tw-bg-primary"
-						>
-							<li>
-								<kv-button variant="ghost" to="#creative-studio-logo-intro">
-									Logos
-								</kv-button>
-							</li>
-							<li>
-								<kv-button variant="ghost" to="#creative-studio-colors-intro">
-									Colors
-								</kv-button>
-							</li>
-							<li>
-								<kv-button variant="ghost" to="#creative-studio-typography-intro">
-									Type
-								</kv-button>
-							</li>
-						</ul>
+							>
```

Comment:
Eventually, It would be useful to drive the nav via the dataObject in Ui Setting from contentful. Each object in the array would have the button text and the href which you could then cycle through to build the nav making it easy to manage.